### PR TITLE
adding network list test

### PR DESCRIPTION
**Pull request by ksouf**
Signed-off-by: khaled souf <khaled.souf@gmail.com>

**- What I did**
adding  network list test 
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cli/command/network/list_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	filterArgs.Add("image.name", "ubuntu")
+
+	expectedOpts := types.NetworkListOptions{
+		filterArgs,
+	}
+
+	cli := test.NewFakeCli(&fakeClient{
+		networkListFunc: func(ctx context.Context, options types.NetworkListOptions) ([]types.NetworkResource, error) {
+			assert.Equal(t, expectedOpts, options, "not expected options error")
+			return []types.NetworkResource{}, nil
```
Comment (dnephin) [inline cli/command/network/list_test.go]: We also print out the returned ID, so we should include an ID here, and check that the ID is printed

File: cli/internal/test/builders/network.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		Scope:      "default scope",
+		Created:    time.Time{},
+		EnableIPv6: false,
+		Attachable: false,
+		Ingress:    false,
+		ConfigOnly: false,
+		Internal:   false,
+		IPAM:       network.IPAM{},
+		ConfigFrom: network.ConfigReference{},
+		Containers: map[string]types.EndpointResource{"toto": {
```
Comment (vdemeester) [inline cli/internal/test/builders/network.go]: This one could be extract in another *builder* function 👼

Comment (dnephin) [inline cli/internal/test/builders/network.go]: I believe many of these fields are already the default "zero" value, so you don't have to include them here either (false, network.IPAM{},etc)

File: cli/internal/test/builders/network.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package builders
```
Comment (dnephin) [inline cli/internal/test/builders/network.go]: This package has moved to `internal/test/builders`, please `git mv` this file

File: cli/command/network/list_test.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+				NetworkResourceDriver("09.7.01"),
+				NetworkResourceScope("global"))}, nil
+		},
+	})
+	cmd := newListCommand(cli)
+
+	cmd.Flags().Set("filter", "image.name=ubuntu")
+	assert.NoError(t, cmd.Execute())
+	assert.Equal(t, "NETWORK ID          NAME                DRIVER              SCOPE\n"+
+		"123454321           network_1           09.7.01             global", strings.TrimSpace(cli.OutBuffer().String()))
```
Comment (dnephin) [inline cli/command/network/list_test.go]: Actually, you might want to use `golden.Assert()` here to make it easier to update in the future. There are a few examples in the repo

**Comment by ksouf** [comment]
@dnephin @vdemeester there is something weird with jenkins build could you check it please :)

**Comment by dnephin** [comment]
looks like jenkins failed to send the build status to github, but the tests passed, so it's all good

**Pull request merged**